extensions/wikia/AssetsManager/config.php

```diff
@@ -215,6 +215,7 @@
 		'//extensions/wikia/AdEngine/js/context/adContext.js',
 		'//extensions/wikia/AdEngine/js/lookup/lookupFactory.js',
 		'//extensions/wikia/AdEngine/js/slot/adSlot.js',
+		'//extensions/wikia/AdEngine/js/slot/adUnitBuilder.js',
```

Comment:
understood, but it can be removed from adengine2_desktop_js because adengine2_tracking_js is added on Oasis